README.md

```diff
@@ -31,6 +31,7 @@ jobs:
 | `failure-level` | The lowest annotation level to fail on ("warning or "error"") | no | `"error"` |
 | `conclusion-level` | Action conclusion ("success" or "failure") if annotations of the failure-level were created. | no | `"success"` |
 | `working-directory` | Which directory to run the action in | no | `"."` |
+| `extensions` | Space separated list of extenions. Only changed files with these extensions will be passed to ESlint. | no | `".js .ts .jsx .tsx .mjs .cjs"` |
```

Comment:
Did you consider a regex pattern for this config? Any particular preference on that vs space separation?

If we allowed regex, we might be able to put something like this. It's definitely not clearer, or simpler, but I wonder if the power would ever be valuable?

`.(m|c)?(j|t)sx?`

Reply:
I did consider it, but I preferred the easier to read space separated version over that. Definitely could be missing something, but I can't immediately think of a solid reason you would want regex over a simple list of extensions.

There might be a _very_ small speed improvement with using that regex, but it would come at the cost of being able to quickly read a list and know if your file _should_ be included in the linter.

Reply:
At the risk of bike shedding here, can I suggest we go a different route: comma-separated list of extensions?  I agree that simple extensions feel easier to use and reason about from both sides (using this action & building/maintaining it).  Since we previously had extensions as a comma separated list, this should make the upgrade even easier without any meaningful difference for us.

Reply:
That's a good point about maintaining compatibility. I like the readability better of space separated, but that change doesn't feel worth the breaking of compatibility I think.